app/src/main/java/nu/yona/app/ui/frinends/TimelineStickyAdapter.java

```diff
@@ -125,6 +109,40 @@ public void onBindViewHolder(final TimelineHolder holder, int position)
 		}
 	}
 
+
+	private void setUpTimeFrameControlHolderView(DayActivity dayActivity, TimelineHolder holder)
+	{
+		if (dayActivity.getTimeZoneSpread() != null)
+		{
+			holder.getmTimeFrameGraph().chartValuePre(dayActivity.getTimeZoneSpread());
+		}
+		updateProfileImage(holder, dayActivity);
+	}
+
+	private void setUpTimeBucketControlHolderView(DayActivity dayActivity, TimelineHolder holder)
+	{
+		int maxDurationAllow = (int) dayActivity.getYonaGoal().getMaxDurationMinutes();
+		if (maxDurationAllow > 0)
+		{
+			holder.getmTimebucketGraph().graphArguments(dayActivity.getTotalMinutesBeyondGoal(), (int) dayActivity.getYonaGoal().getMaxDurationMinutes(), dayActivity.getTotalActivityDurationMinutes());
+		}
+		updateProfileImage(holder, dayActivity);
+	}
+
+	private void setUpNoGoHolderView(DayActivity dayActivity, TimelineHolder holder)
+	{
+		if (dayActivity.getGoalAccomplished())
+		{
+			holder.getmNogoImage().setImageResource(R.drawable.adult_happy);
```

Comment:
Some day, you should rename these. It should be ``no_go_happy`` and ``no_go_sad``.

Reply:
I will note this in the 1.2 refactoring ticket